setup/ipc-meta-setup.sh

```diff
@@ -50,7 +50,7 @@ ls -1 "${BASEDIR}"/[0-9]*.sh | sort | while read SCRIPT; do
         echo "SKIP: ${SCRIPT_NAME} has already succeeded before"
     else
         echo "APPLY: running ${SCRIPT_NAME}..."
-        ./${SCRIPT} || die "${SCRIPT}"
+        ${SCRIPT} || die "${SCRIPT}"
```

Comment:
Ok, taking this point back - we did exec so previously IIRC, but not anymore as I see now from context.